### ISSUE TITLE
Grid drawn below other windows in some cases

### ISSUE THREAD

**Issue by lesderid** [lesderid/keynavish#10]
If the process launching keynavish isn't foregrounded (i.e. doesn't own the window returned by `GetForegroundWindow()`), keynavish will be drawn below other windows.

We can't simply foreground the grid window, as this breaks `windowzoom` and causes the foreground window to be deactivated (which we may not want).

Another possible solution would be to set the foreground window to the grid window and then reset it back to the original one:
```d
HWND originalWindow = GetForegroundWindow();
SetForegroundWindow(keynavishWindow);
SetForegroundWindow(originalWindow);
```

However, this seems to be visually glitchy.

**Comment by lesderid** [comment]
This issue seems to be more frequent on Windows Server 2019 (so probably Windows 10 too) than on Windows 8.1. Suggestions very welcome.

**Comment by lesderid** [comment]
This seems to have been fixed by f4dd77953e65f15d3920a89958f57eef9362676d, but I'm not sure if it works in all cases.
